pkgs/tools/networking/opensnitch/daemon.nix

```diff
@@ -40,9 +40,12 @@ buildGoModule rec {
 
   nativeBuildInputs = [ pkg-config protobuf go-protobuf makeWrapper protoc-gen-go-grpc ];
 
-  vendorSha256 = null; #vendorSha256 = "";
+  vendorSha256 = "sha256-jWP0oF+jZRFMi5Y2y0SARMoP8wTKVZ8UWra9JNzdSOw=";
 
   preBuild = ''
+    # Fix inconsistent vendoring build error
```

Comment:
Do we have an upstream issue reference for this?

Reply:
Okay added an upstream bug report